wiki/Guides/How_To_Get_Your_Map_Ranked/en.md

```diff
@@ -6,19 +6,19 @@ outdated: true
 
 *See also: [Ranking Criteria](/wiki/Ranking_Criteria)*
 
-Having trouble getting your [map](/wiki/Glossary) ranked? You've come to the right place.
+Having trouble getting your [beatmap](/wiki/Glossary) ranked? You've come to the right place.
 
 ## Step 1: Modding
 
 ***Precursor: [Getting Your Map Modded](/wiki/Modding_Guides/Getting_Your_Map_Modded)***
 
-The first and most important step towards getting your map ranked is to get people looking at it - people willing to [mod](/wiki/Modding) your map can be found in \#modreqs, the modding queues subforum or by asking them directly after checking their signature for modding policies. Remember that all mods are useful - any perspective on your map can be used to make it a better one. Consider every suggestion a modder makes, and if you turn any down, post a detailed response in the map thread making it clear the reasons why. It makes them a better modder and makes you think more about how your map can improve! Also, if you found the mod particularly useful, be sure to give the post a [kudosu](/wiki/Glossary).
```

Comment:
> be sure to give the post a [kudosu](/wiki/Glossary).
I would say to change this to  "be sure to upvote the post!"

Reply:
Changed it to say ‘give the post an upvote’